packages/ui/src/components/Document/actions/DeleteMappingItemAction.tsx

```diff
@@ -44,16 +51,17 @@ export const DeleteMappingItemAction: FunctionComponent<DeleteItemProps> = ({ no
         title={title}
         onClose={closeModal}
         data-testid="delete-mapping-modal"
-        actions={[
+      >
+        <ModalHeader title={title} />
+        <ModalBody>{title}?</ModalBody>
```

Comment:
Do we need the `question` mark?

Reply:
It was there previously in the modal, so I left it there. It sounds more like a question, so I think it's OK to leave it there.